requirements.txt

```diff
@@ -1,4 +1,5 @@
 # test
+marshmallow
```

Comment:
don't need to install this because when webargs is installed https://github.com/teracyhq/flask-classful/blob/develop/.travis.yml#L38, it will be installed, too.

Reply:
Thanks, commit amended.